### DISCUSSION TITLE
I don't think this is working

### DISCUSSION THREAD

**Discussion by tonyennis** [imartinez/privateGPT#722] [General]
The destructions say it should take 30 seconds per file to create embeddings .  My installation is consuming several files per second, and the results coming from privateGPT are nonsense.  What could be wrong here?

**Comment by sime2408** [comment]
You have to paste your most important parts of the `.env` file here, for the people to help you, your OS, your hardware...

**Comment by tonyennis** [comment]
![image](https://github.com/imartinez/privateGPT/assets/7684843/7b78d31f-9acd-4954-a71e-17fdc3b7f600)

Here's the env file. I am not the installer, but I believe it is stock.

**Comment by tonyennis** [comment]
Something that confuses me is that the embeddings are downloaded "the first time", if I understand correctly.  If so, where would it be stored?

**Comment by sime2408** [comment]
nonsense results usually mean that you're either using a weak model or your files are having hardly readable content for the embeddings like pdf files which are not OCR-ed properly and so on. The embeddings model is downloaded under your users' profile home directory .cache/huggingface folder (at least last time I checked on my Mac)

**Reply by tonyennis** [reply]
I'm on an ubuntu system. The embeddings file is not under the chroma or huggingface folders I found there.
